src/index.html

```diff
@@ -11,7 +11,174 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="form_fieldSet">
```

Comment:
You cant combine camelCase and snake_case, stick with one of them, I'd suggest snake case for classes names
```suggestion
      <fieldset class="form_fieldset">
```